mainnet.txt

```diff
@@ -2,4 +2,11 @@
 cdcf936d70726dd724e0e6a8353d8e5ba5abdd20@seed2.mainnet.pokt.network:26663
 74b4322a91c4a7f3e774648d0730c1e610494691@seed3.mainnet.pokt.network:26662
 b3235089ff302c9615ba661e13e601d9d6265b15@seed4.mainnet.pokt.network:26663
+ab5776d1c9a50d47e30e90a0600e787d9fb74643@seed5.mainnet.pokt.network:26664
+a6dec84acfe1bae887cd6c4f2eeb864fae29c9f7@seed6.mainnet.pokt.network:26664
 
+Also you can use these seeds if the list above fails for any reason:
```

Comment:
Can you remove these lines 8-12?  They're duplicated.

Reply:
Done